cmake/jana_plugin.cmake

```diff
@@ -176,7 +176,7 @@ macro(plugin_add_acts _name)
 
     if(NOT Acts_FOUND)
         find_package(Acts REQUIRED COMPONENTS Core PluginIdentification PluginTGeo PluginJson PluginDD4hep)
-        set(Acts_VERSION_MIN "19.0.0")
+        set(Acts_VERSION_MIN "20.2.0")
```

Comment:
20.2 (versus 20.0) is a hard requirement?

Reply:
Yeah, acts-20.2.0 introduced this change:
```diff
  std::optional<MultiTrajectory> m_multiTrajectory;
  std::shared_ptr<MultiTrajectory> m_multiTrajectory;
```
The way they get around this in a minor release is that technically this is a change in their examples, not their main interfaces.

Reply:
Ref: https://github.com/acts-project/acts/pull/1507